### PR TITLE
[feature] Add compatibility to not lts 2.06 (2.06+)

### PR DESCRIPTION
**Pull request by Shnoulle**
- Replace throw error by an alert
- Add a button in "plugin" settings for surveys

I use floatval(version) >= 2.5 , maybe we need more control with build number, tested in last 2.06+

### CODE REVIEW

File: MassAction.php [hunk 1]
```diff
@@ -25,32 +25,53 @@ public function init()
     {
         $config = require(Yii::app()->basePath . '/config/version.php');
         $this->lsVersion = $config['versionnumber'];
-
-        if ($this->lsVersion == '2.5')
+        // Add build number ?
+        if (floatval($this->lsVersion) >= '2.5')
```
Comment (olleharstedt) [inline MassAction.php:29]: Right-side should be `2.5`, not a string.

File: MassAction.php [hunk 2]
```diff
@@ -700,13 +721,13 @@ public function newDirectRequest()
         if ($event->get('target') == "MassAction")
         {
             $request = $event->get('request');
-            $functionToCall = $event->get('function'); 
+            $functionToCall = $event->get('function');
             // TODO: Hardcode functions
-            if ($this->lsVersion == '2.5' || $functionToCall != "actionIndex")
+            if (floatval($this->lsVersion) >= '2.5' || $functionToCall != "actionIndex")
```
Comment (olleharstedt) [inline MassAction.php:726]: Same here (2.5 as string).

**Comment by olleharstedt** [comment]
Looks good! Which version did you test on? 2.05, 2.06?

Oh, you wrote 2.06+. Got it.

**Comment by olleharstedt** [comment]
Merged. I'll fix those small string stuff.

**Comment by Shnoulle** [comment]
Thanks , silly for string. But comparing with == then it work ;)

I do a quick test with last 2.05 :) and send the info on README ;)

**Comment by olleharstedt** [comment]
Great!

**Comment by Shnoulle** [comment]
Snif ..... LS 2.05+ / build 150520 : installer : InstallerController and its behaviors do not have a method or closure named "return_bytes".

I think i don't want to find the good 2.05 version ;) Maybe some of my dev testing somewhere if i found one.

**Comment by olleharstedt** [comment]
2.05 is not a requirement. Don't waste any time on it.

**Pull request merged**